src/server/CodeStyle/CodeStyle.py

```
@@ -7,51 +7,54 @@
-
-    return formatted_code, errors
+class CodeStyleFormatter:
+    def __init__(self, config_path=None):
+        self.configs = ConfigClass(config_path) if config_path else None
+
+    def load_config(self, config_path):
+        self.configs = ConfigClass(config_path)
+        return self.configs
+
+    def parse_java_code(self, code):
+        lexer = JavaLexer(InputStream(code))
```

Ruff (0.11.9)

`InputStream` may be undefined, or defined from star imports

(F405)